documentation/grouping.html

```diff
@@ -1533,18 +1533,21 @@ <h3 id="time-and-date">Time and date</h3>
 <h3 id="counting-unique-groups">Counting unique groups</h3>
 <p>
 The <code>count</code> aggregator can be applied on list of groups to determine the number of unique groups
-without having to explicitly retrieve all groups.
+without having to explicitly retrieve all groups. Note that that this count is an estimate using HyperLogLog++ which is an algorithm for the count-distinct problem. 
+To get an accurate count one needs to explicitly retrieve all groups and count them in a custom component or in the middle tier calling out to Vespa. This is network intensive and might not be feasible in cases with many unique groups.
+
 Another use case for this aggregator is counting the number of unique instances matching a given expression.
 
-Output the number of groups, which is equivalent to the number of unique values for attribute "a":
+
+Output an estimate of the number of groups, which is equivalent to the number of unique values for attribute "a":
 <pre>all( group(a) output(count()) )</pre>
-Output the number of unique string lengths for the attribute "name":
+Output an estimate of the number of unique string lengths for the attribute "name":
 <pre>all( group(strlen(name)) output(count()) )</pre>
-Output the sum of the "b" attribute for each group in addition to the overall group count:
+Output the sum of the "b" attribute for each group in addition to an estimated count of the overall number of unique groups:
```

Comment:
It's not an estimate in this case. Since the each clause causes all groups to be returned, the estimated value is replaced by the actual group count.

Reply:
Addressed in 279d71e